### PR TITLE
Inserting labels for the signin button

### PR DESCRIPTION
**Pull request by filipeotero**
It's included in this PR the definition for the status of the 'Sign In' button

@QilongTang @avidit @zeusongit

### CODE REVIEW

**Review by avidit — APPROVED** [review]

File: src/Static.js [hunk 1]
```diff
@@ -193,15 +191,19 @@ class Static extends React.Component {
 
 Static.defaultProps = {
   signInTitle: 'Sign in',
+  signingInTitle: 'Siging in',
```
Comment (zeusongit) [inline src/Static.js:194]: spelling mistake : `Signing In`

File: src/Static.js [hunk 2]
```diff
@@ -193,15 +191,19 @@ class Static extends React.Component {
...
+  signOutTitle: 'Sign out',
   launchTitle: 'Launch Dynamo',
   showScreenAgainLabel: 'Don\'t show this screen again',
   importSettingsTitle: 'Import Settings',
   importSettingsTooltipDescription: 'You can import custom settings here, which will overwrite your current settings. If you\'d like to preserve a copy of your current settings, export them before importing new settings. Settings not shown in the Preferences panel will be applied once Dynamo and any host program restarts.'
-
+
```
Comment (zeusongit) [inline src/Static.js:200]: remove spaces

File: src/Static.js (hunk 1 above)
Comment (zeusongit) [reply src/Static.js:194]: Also Capitalize `In` in both Sign In and Signing In

**Review by zeusongit — APPROVED** [review]
2 comment, other than that LGTM

**Comment by zeusongit** [comment]
Merging, Please Cherry pick

**Pull request merged**